diff --git a/nbgrader/nbextensions/assignment_list/assignment_list.js b/nbgrader/nbextensions/assignment_list/assignment_list.js
index 523fc99..56de43a 100644
--- a/nbgrader/nbextensions/assignment_list/assignment_list.js
+++ b/nbgrader/nbextensions/assignment_list/assignment_list.js
@@ -118,6 +118,7 @@ define([
             this.default_course_element.text(course);
         }
         this.current_course = course;
+        this.default_course_element.text(this.current_course);
         var success = $.proxy(this.load_assignment_list_success, this);
         this.assignment_list.load_list(course, success);
     };
@@ -139,7 +140,6 @@ define([
             this.data = undefined;
         }
 
-        this.default_course_element.text(this.current_course);
         this.enable_list();
     };
 
diff --git a/nbgrader/nbextensions/assignment_list/main.js b/nbgrader/nbextensions/assignment_list/main.js
index f135220..f327fc2 100644
--- a/nbgrader/nbextensions/assignment_list/main.js
+++ b/nbgrader/nbextensions/assignment_list/main.js
@@ -1,7 +1,9 @@
-define(function(require) {
-    var $ = require('jquery');
-    var Jupyter = require('base/js/namespace');
-    var AssignmentList = require('./assignment_list');
+define([
+    'base/js/namespace',
+    'jquery',
+    './assignment_list'
+], function(Jupyter, $, AssignmentList) {
+    "use strict";
 
     var assignment_html = $([
         '<div id="assignments" class="tab-pane">',
diff --git a/nbgrader/tests/nbextensions/test_assignment_list.py b/nbgrader/tests/nbextensions/test_assignment_list.py
index 78c4f05..0fe50ae 100644
--- a/nbgrader/tests/nbextensions/test_assignment_list.py
+++ b/nbgrader/tests/nbextensions/test_assignment_list.py
@@ -90,7 +90,8 @@ def _sort_rows(x):
 
 
 def _wait_until_loaded(browser):
-    _wait(browser).until(lambda browser: browser.find_element_by_css_selector("#course_list_dropdown").is_enabled())
+    _wait(browser).until(lambda browser: browser.find_element_by_id("course_list_default").text != "Loading, please wait...")
+    _wait(browser).until(EC.element_to_be_clickable((By.ID, "course_list_dropdown")))
 
 
 def _change_course(browser, course):
diff --git a/nbgrader/tests/nbextensions/test_create_assignment.py b/nbgrader/tests/nbextensions/test_create_assignment.py
index a493e35..091cb44 100644
--- a/nbgrader/tests/nbextensions/test_create_assignment.py
+++ b/nbgrader/tests/nbextensions/test_create_assignment.py
@@ -117,17 +117,10 @@ def _save(browser):
 
 def _save_and_validate(browser):
     _save(browser)
-
-    # wait for notebook_saved.Notebook event
-    def notebook_saved(browser):
-        elem = browser.find_element_by_css_selector("#notification_notebook")
-        if not elem.is_displayed():
-            return False
-        if elem.text == "Notebook saved":
-            return True
-        return False
-
-    _wait(browser).until(notebook_saved)
+    _wait(browser).until(EC.text_to_be_present_in_element(
+        (By.ID, "notification_notebook"),
+        "Notebook saved"
+    ))
     read("blank.ipynb", current_nbformat)
 
 
